README.md

```diff
@@ -78,7 +78,9 @@ fluorophores_table = FluorophoresTable(description="fluorophores")
 fluorophores_table.add_row(
     label="dlight",
     location="VTA",
-    coordinates=(3.0,2.0,1.0)
+    coordinates=(3.0,2.0,1.0),
+    excitation_lambda=700.0,
+    emission_lambda=500.0
```

Comment:
I agree with keeping `emission_peak_wavelength` and `excitation_peak_wavelength` properties here, besides this small edit I think this PR is good to go. 

```suggestion
    excitation_peak_wavelength=700.0,
    emission_peak_wavelength=500.0
```